src/test/resources/hw1/testSuite.xml

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+<suite name="calculator's methods test">
+
+    <test name="SumSubCalculatorTest">
```

Comment:
Instead of enumerating all classes, `package` can be referred

one class is missed